recipe/meta.yaml

```diff
@@ -1,6 +1,7 @@
 {% set name = "telemac-mascaret" %}
-{% set version = "v8p4r0" %}
+{% set version = "v8p5" %}
```

Comment:
V8p5 is not officially released 
https://gitlab.pam-retd.fr/otm/telemac-mascaret/-/releases
I would suggest to wait (commit can still be cherry-picked)

Reply:
weird. They've announced it at the conference. I will engage @sebourban on slack
anyway this is minor 
ok